python-phoenixdb/NEWS.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Version 1.0.1
+-------------
+
+- Use HTTP sessions to enable sticky load balancers
+- Revert default GSSAPI OID to "1.3.6.1.5.5.2" to improve compatibility
```

Comment:
Is this generated through some script or manually? If manually, can we also add Jira number?
I have not used `.rst` format so far, so I might be wrong about the convention followed :)